.github/workflows/gh-pages.deploy.yml

```diff
@@ -25,6 +21,12 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
 
+      - name: Load NEXT_PUBLIC_URL and NEXT_PUBLIC_BASE_PATH from .env
+        run: |
+          set -o allexport
+          source .env
+          set +o allexport
+
```

Comment:
We also need to make sure basePath is set for `dev` in the next.config.js file at the root. This will make so a user does not need to navigation to http://localhost:5001/canopy-iiif to view things in a browser.

```
const basePath = isDev ? `` : process.env.NEXT_PUBLIC_BASE_PATH;
```